packages/plugin-equations/src/editor.tsx

```diff
@@ -74,13 +74,13 @@ export function EquationsEditor(
 
   const { focused, state, editable } = props
   const children = R.flatten(
-    props.state.steps().map(step => {
+    Array.from(props.state.steps).map(step => {
```

Comment:
Since `list` is only ArrayLike, we need `Array.from` if we want to use Array methods (like `Array.map`). We could provide a getter `.items` that already provides a "real" array

Reply:
Fixed